src/azul/openapi/schema.py

```diff
@@ -45,7 +46,7 @@ class optional(NamedTuple):
 
 
 # noinspection PyShadowingBuiltins
-def object(additional_properties=False, **props: Union[TYPE, optional]):
+def object(additional_properties=False, **props: Union[TYPE, optional]) -> JSON:
```

Comment:
Unless there is precedent for return values typed JSON in this module, this really should be MutableJSON. See C guide. If there is precedent, open ticket to fix consistently.

Reply:
There is precedent for returning `JSON`:
https://github.com/DataBiosphere/azul/blame/072af3b9bd14398b8820a94c50cb8c4ed4235cae/src/azul/openapi/schema.py#L141
https://github.com/DataBiosphere/azul/blame/072af3b9bd14398b8820a94c50cb8c4ed4235cae/src/azul/openapi/schema.py#L250
https://github.com/DataBiosphere/azul/blame/072af3b9bd14398b8820a94c50cb8c4ed4235cae/src/azul/openapi/schema.py#L277

I added this change to the schema.helpers PR. https://github.com/DataBiosphere/azul/issues/2604